packages/app/src/components/Randomizer/Randomizer.js

```diff
@@ -24,6 +24,7 @@ const Randomizer = ({
   maxTimeBetweenRolls = 1000,
   resultWaitTime = 1250
 }) => {
+  let resId = null
```

Comment:
What is the purpose of this? Why not `restaurant.id` and `restaurants[i].id`?

Reply:
I couldn't make restaurant.id to work. The restaurants[index].id is a solution that either required an if-statement before restaurantService.increaseResultAmount(resId) to check if restaurants.length === 1, to see if I should use restaurants[0] or restaurants[roll % restaurants.length] so it was either one more if-statement or one more variable. Since variable also fixes the restaurant.id problem, I decided to go with that.